content/_guides/clients.md

```diff
@@ -39,12 +39,12 @@ These are clients that you navigate primarily with your mouse cursor or touch.
 Desktop clients:
 
 - [Halloy](https://github.com/squidowl/halloy) - Windows, Linux, macOS (free)
-- [Hexchat](https://hexchat.github.io/) - Windows, Linux (free)
```

Comment:
IMO, hexchat is still a fine client to use. it still works, it's one of the few clients available on windows, and it's pretty popular